content/getting-started/linux.md

```diff
@@ -45,7 +47,23 @@ There are some additional requirements to compile TinyGo programs that can run o
 
 #### ARM Cortex-M
 
-To compile TinyGo programs for ARM based processors you must install CLang 8 (`clang-8`) for building assembly files and the compiler runtime library [https://compiler-rt.llvm.org/](https://compiler-rt.llvm.org/).
+To compile TinyGo programs for ARM based processors you must also install Clang 8 (`clang-8`) for building assembly files and the compiler runtime library [https://compiler-rt.llvm.org/](https://compiler-rt.llvm.org/).
+
+One way to do this is to use the LLVM Debian/Ubuntu packages.
+
+First, you must add the LLVM package repository to your system. Note that if you have already done this once, you do not need to do it again.
+
+```shell
+wget -O - https://apt.llvm.org/llvm-snapshot.gpg.key|sudo apt-key add -
+sudo add-apt-repository "deb http://apt.llvm.org/$(lsb_release -s -c)/ llvm-toolchain-$(lsb_release -s -c)-8 main"
+sudo apt-get update
+```
+
+Now you can install Clang and the LLD linker by running this command:
+
+```shell
+sudo apt-get install clang-8 lldb-8 lld-8
```

Comment:
Why install `lldb-8`? That's a debugger, not a linker. What you really need is something like:

```
sudo apt-get install clang-8 llvm-8-dev lld-8 libclang-8-dev
```

Reply:
Done.